src/Resources/scaffolds/6.0/bootstrapcss/assets/styles/app.css.tpl

```diff
@@ -0,0 +1 @@
+@import "~bootstrap/dist/css/bootstrap.css";
```

Comment:
This is a spot where, like with `app.js`, I'd love to patch instead of replace. There's a practical reason. Right now, when you run the bootstrapcss scaffold, (A) the recipe adds `bootstrap.css` and then (B) the scaffold tries to replace it... so it immediately asks us if we want to overwrite `app.css`... which is weird :p.

Reply:
I was wondering that too. Ignore it? As I, keep it there (and just add the import for bootstrap). It doesn't really hurt anything. I added that background just so there was SOMETHING, and then you'd think "hey, where does that come from". I think it still accomplishes that.